### ISSUE TITLE
MongoError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by granteagon** [balderdashy/sails-mongo#342]
I'm getting this error on the latest everything. Whenever I try to update a record, whether via ajax PUT or sails console.  Please help!  I've been trying to solve for two days and I'm at a loss.

The error:
`MongoError: Maximum call stack size exceeded`

Versions I'm running:
```zsh
backend feat-dashboard-frontend % sails --version
0.11.2
backend feat-dashboard-frontend % npm --version
3.3.3
backend feat-dashboard-frontend % node --version
v4.0.0
backend feat-dashboard-frontend % mongod --version
db version v3.0.6
backend feat-dashboard-frontend % cat node_modules/sails-mongo/package.json | grep sails-mongo | head -1
    "sails-mongo": "0.11.4",
```

**Comment by sailsbot** [comment]
Thanks for posting, @granteagon.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
